tests/app/views/test_suppliers.py

```diff
@@ -1350,6 +1316,38 @@ def test_get_supplier_framework_info(self):
             'uploaderUserId': 20
         }
 
+
+class TestSupplierFrameworkUpdates(BaseApplicationTest, JSONUpdateTestMixin):
+    # These variables are for the JSONUpdateTestMixin, which checks we get standardised
+    # error messages when we POST bad JSON to this route (the SupplierFramework updates route)
```

Comment:
Sorry, thanks for the explanation but it probably doesn't need to actually be included as a comment (at least not here)